src/tests/ftest/datamover/dm_large_file.yaml

```diff
@@ -43,8 +43,8 @@ ior:
       transfer_size: '1M'
       block_size: '5G'        # aggregate of 150G for dcp and 50G for fs_copy
   objectclass:
-    SX:
-      dfs_oclass: "SX"
+    EC_4P1GX:
+      dfs_oclass: "EC_4P1GX"
```

Comment:
We'll have to see if this passes, but I think we might still have space imbalance, since the container has the default chunk size and ec cell size of 1M, but we're using 4P1. If we still run out of space, we might need to adjust ec cell size and/or chunk size. Similar to what was discussed at the last Perf WF meeting, right @ravalsam  ?

Reply:
We have 8 targets in Ci and the aggregated size is 150G with 10 processes per CN. So GX might be ok but let's see because it's a single shared file.

Reply:
Both test passed (y)